apps/server/src/modules/tool/tool.module.ts

```diff
@@ -4,13 +4,15 @@ import { Lti11ResponseMapper } from '@src/modules/tool/mapper/lti11-response.map
 import { LoggerModule } from '@src/core/logger';
 import { LtiToolService } from '@src/modules/tool/service/lti-tool.service';
 import { SchoolModule } from '@src/modules/school';
+import { ToolController } from '@src/modules/tool/controller/tool.controller';
```

Comment:
Please split it to tool-api.module.ts and tool.module.ts
For example look  Lesson Module

Reply:
Is this already set as the target-architecture? Afaik we did not yet decide on the final structure.

If this is the final structure, then i would like to have do the separation in a follow-up ticket, since we will have further tasks here. Separating them here will increase the complexity of this even more.